packages/flutterfire_ui/lib/src/firestore/table_builder.dart

```diff
@@ -58,12 +70,24 @@ class FirestoreDataTable extends StatefulWidget {
     this.dragStartBehavior = DragStartBehavior.start,
     this.arrowHeadColor,
     this.checkboxHorizontalMargin,
+    this.cellBuilder,
+    this.canEditCell = true,
+    this.onEditItem,
   })  : assert(
           columnLabels is LinkedHashMap,
           'only LinkedHashMap are supported as header',
         ), // using an assert instead of a type because `<A, B>{}` types as `Map` but is an instance of `LinkedHashMap`
         super(key: key);
 
+  /// When specified, the builder will be use to disply your own widget for the cel
+  final CellBuilder? cellBuilder;
+
+  /// When set to false this allow row selection, defaults is set to true
```

Comment:
```suggestion
  /// When set to false this allows row selection, defaults to true
```

Reply:
Why is the comment about "selection" but the name is about edits?

Reply:
I was trying to not break the current API and allow row selection without triggering editing a cell

I will change the name to reflect more what's happening.